dub.json

```diff
@@ -16,6 +16,10 @@
         {
             "name": "dynamic",
             "targetType": "dynamicLibrary",
+        },
+        {
+            "name": "unittest",
+            "versions": ["to_be_filled"],
```

Comment:
this is added, s.t. the version string can be conveniently changed, e.g. by the CI or user
(I would prefer if  DUB to allowed this via the CLI though)